how-to/workspace-platform-starter/public/platform/provider.html

```diff
@@ -7,6 +7,7 @@
 		<link rel="icon" type="image/x-icon" href="../favicon.ico" />
 		<meta name="description" content="" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<link rel="stylesheet" href="../common/style/app.css" />
```

Comment:
Why add styling to the provider? Make it look nicer?

Reply:
Ah interesting one that, its because I add a temporary dummy canvas to the provider to measure how wide the text for the menus it going to be, and I need the correct app fonts and styling to be used when measuring.